acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -0,0 +1,86 @@
+@parallel=false
+# for https://folio-org.atlassian.net/browse/MODORDSTOR-402
+Feature: Open ongoing order
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(centralTenant)', 'Accept': 'application/json' }
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+    * def orderId = callonce uuid3
+    * def orderLineIdOne = callonce uuid4
+
+  Scenario: Create fund and budget
+    * call createFund { 'id': '#(fundId)' }
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 100, 'fundId': '#(fundId)', 'status': 'Active' }
+
+  Scenario: check budget after create
+    Given path '/finance/budgets'
+    And param query = 'fundId==' + fundId
+    When method GET
+    Then status 200
+
+  Scenario: Create orders
+
+    Given path 'orders/composite-orders'
+    And request
+    """
+    {
+      id: '#(orderId)',
+      vendor: '#(globalVendorId)',
+      orderType: 'Ongoing',
+      "ongoing" : {
+        "interval" : 123,
+        "isSubscription" : true,
+        "renewalDate" : "2022-05-08T00:00:00.000+00:00"
+      }
+    }
+    """
+    When method POST
+    Then status 201
+
+  Scenario Outline: Create order lines for <orderLineId> and <fundId>
+    * print 'college >>> ' + collegeTenant
+    * def orderId = <orderId>
+    * def poLineId = <orderLineId>
+
+    Given path 'orders/order-lines'
+
+    * def orderLine = read('classpath:samples/consortia/multi-tenant-order-line.json')
+    * set orderLine.id = poLineId
+    * set orderLine.purchaseOrderId = orderId
+    * set orderLine.cost.listUnitPrice = <amount>
+    * set orderLine.fundDistribution[0].fundId = <fundId>
+    * set orderLine.locations[2].tenantId = collegeTenant
+
+    And request orderLine
+    When method POST
+    Then status 201
+
+    Examples:
```

Comment:
If possible, please use Tables instead of Scenario Outlines & Examples in newer tests, because I understand refactoring it now may take too much time

Reply:
Extracted existing reusable block as a separate functions. It's possible to farther improve them but it makes more sense to do this in scope of a separate ticket